### ISSUE TITLE
An Apple a Day

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#2451]
**Completed Reports**

> You can store apple cider even if you don't have any.

**Comment by Bennylavaa** [comment]
yep you dont have to collect anything at all just use the storing object and done, should be considered high prio as it makes most this quest pointless.
